CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Code of COnduct is exactly taken from https://www.contributor-covenant.org/version/2/1/code_of_conduct/